src/devhub/entity/community/Compose.jsx

```diff
@@ -0,0 +1,477 @@
+if (!context.accountId) {
+  return <></>;
+}
+
+const optimisticUpdateFn = props.optimisticUpdateFn;
+const clearOptimisticUpdateFn = props.clearOptimisticUpdateFn;
+
```

Comment:
Where did these two functions come from? It looks like optimisticUpdate is for immediately reflecting the newly created post?

Reply:
yeah, we can use it for passing some custom function for commit and cancel as well